bin/data/rename_category

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require_relative "../../config/environment"
+require_relative "../../config/cli"
+
+@cli = CLI.new([:name, :id]) do |opts|
+  opts.on("-n NAME", "--name=NAME", String, "New name for the category")
+  opts.on("-i ID", "--id=ID", String, "ID of the new category")
+end
+@cli.parse!(ARGV)
+@cli.options_status
+
+fail "Missing required option: --name" unless @cli.options.name
+fail "Missing required option: --id" unless @cli.options.id
+
+category = Category.find_by(id: @cli.options.id)
+fail "Category `#{@cli.options.id}` not found" unless category
+
+original_name = category[:name] # skip localizations
```

Comment:
Seems redundant. Looks to me like this script only works on English anwyays

Reply:
This is more for active dev than _running as normal_. Seeing the raw value is helpful here IMO